frontend/src/App.css

```diff
@@ -1,6 +1,6 @@
 #app {
     display: grid;
-    grid-template-columns: 50% 50%;
+    grid-template-columns: 25% 75%;
```

Comment:
Wouldn't it be better to split the `EntityDetails` component into 2 parts, one with metadata and history, the other with the helpers, and make this a grid of `25% 50% 25%`? `Helpers` would become a container module (connected to the store), which might also make the code a bit simpler.

This is absolutely a question btw, if you think it's not a good idea or is too hard, feel free to say so. :)

Reply:
I was thinking about that and decided for the current implementation, because it made switching between the single and batch editor slightly easier.

Reply:
Good point!

Reply:
Good point!